readthedocs/organizations/views/base.py

```diff
@@ -72,7 +72,7 @@ def get_organization_queryset(self):
         """
         if self.admin_only:
             return Organization.objects.for_admin_user(user=self.request.user)
-        return Organization.objects.for_user(user=self.request.user)
+        return Organization.objects.public(user=self.request.user)
```

Comment:
We don't have the concept of privacy level in Organizations, so I'm not sure these changes make a ton of sense.